### ISSUE TITLE
Delirious improvement

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1485]
It could be like Everything Jar where you pick a boss for yourself

At first I didn't like it too much because it'd make Everything Jar stand out less with its mechanic but then again we already have Larynx so

the bosses could stay with you through the rooms but maybe their HP should deteriorate overtime to not be too busted

**Comment by Turtier** [comment]
Yeah, mega blast and mega mush are both 12 room charge items last between rooms for a set time, and then delirious has both a much weaker effect than those two and doesn't even last between rooms. I thought bosses lasting between rooms would be a good buff, but your exactly right actually, either have hp drain over time or they instantly die after 15-30 seconds or something. Additionally, another change that could make sense, is that the boss summoned is always a boss from the current chapter, so you only get first floor bosses on first floor and so on.

Just had an extra idea... not even sure if its a good one, but I think its funny, maybe the bosses spawned from delirious drop a boss item on death.

**Comment by Rchardon** [comment]
Delirious was improved in rep+

**Comment by Lonrite** [comment]
> Delirious was improved in rep+

wouldn't say so personally lol. If anything it got nerfed imo & the change wasn't exactly what I wanted but ah well
